src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -1160,3 +1243,63 @@ def blobs_bucket():
 
   assert not environment.get_value('PY_UNITTESTS')
   return local_config.ProjectConfig().get('blobs.bucket')
+
+
+def uworker_io_bucket():
+  """Returns the bucket where uworker I/O is done."""
+  test_uworker_io_bucket = environment.get_value('TEST_UWORKER_IO_BUCKET')
+  if test_uworker_io_bucket:
+    return test_uworker_io_bucket
+
+  assert not environment.get_value('PY_UNITTESTS')
+  # TODO(metzman): Use local config.
+  return environment.get_value('UWORKER_IO_BUCKET')
```

Comment:
Do we need to log an error if this `UWORKER_IO_BUCKET` is undefined?

Reply:
Done.